_includes/overall-schedule.md

```diff
@@ -32,8 +32,21 @@ Organizers will inform participants of the week schedule by email.
   "@id": "{{ site.url }}/{{ cohort }}/schedule",
   "@type": "Course",
   "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
-  "description": "{%- for w in schedule.weeks %} {%- capture w-desc %}**Week {{ w[0] }}** (start. {{ w[1].start }}){% endcapture %} {%- for c in w[1].calls %}{%- capture date %} {% if c.type !='Mentors-Mentee'%} {{ c.date }} {% if c.time %} ([{{ c.time }} European Time] (https://arewemeetingyet.com/Berlin/{{ c.date | date: '%Y-%m-%d' }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))) {% endif %}{% endif %}{% endcapture %} | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: '(' remove: | remove:')'| remove: '@' | remove: ':' | remove: ',' | replace: ' ', '-' | remove: '&'}}-calls) | {{ date }} | [**{{ c.title }}**] (/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |{%- assign w-desc = '' %} {%- endfor %} {%- endfor %}",
+  "description":"{{site.description}}" ,
     "hasCourseInstance": [
+     {%- for w in schedule.weeks %} {%- for c in w.calls %} {%- endfor %} {%- endfor %}
```

Comment:
The issue there is that you close the for loops ( `endfor`) before using the content of it.
You need to move both `endfor` to line 60

Reply:
Oh! That explains why. Thanks so much @bebatut 
I did just that, please take a look when you free. Thanks